pkg/adapter/mtping/pingsource_test.go

```diff
@@ -210,6 +215,7 @@ func TestAllCases(t *testing.T) {
 					rtv1.WithPingSourceSink(sinkAddr),
 					rtv1.WithPingSourceCloudEventAttributes,
 					rtv1.WithPingSourceDeleted,
+					rtv1.WithPingSourceOIDCIdentityCreatedSucceededBecauseOIDCFeatureDisabled(),
```

Comment:
@Leo6Leo could you add a unit test where this is not disabled here?

Reply:
Do we actually need to test OIDC here? Because we have already tested the case when OIDC failed in the pingsource test, I think it might be extra to test it again here in the mt-adapter section.

And we didn't change anything in the mt-adaptor, so I think we wouldn't need to add the unit test here. WDYT cali? @Cali0707